src/components/file/File.js

```diff
@@ -362,7 +362,13 @@ export default class FileComponent extends Field {
         webViewCamera.getPicture((success) => {
           window.resolveLocalFileSystemURL(success, (fileEntry) => {
               fileEntry.file((file) => {
-                this.upload([file]);
```

Comment:
@travist @oneserve-alh 

Will below change work ?
Wouldn't "this" in onloadend function be assigned to FileReader context ?

Reply:
You are right, the scope isn't correct. I'll verify and change to;

```
  reader.onloadend = (evt) => {
      var blob = new Blob([new Uint8Array(evt.target.result)], { type: file.type });
```